### ISSUE TITLE
META: migrate urbit/urbit issues

### ISSUE THREAD

**Issue by ohAitch** [urbit/arvo#13]
Anything that [isn't directly a vere issue](https://github.com/urbit/urbit/issues?q=-label:u3+is:open+is:issue) should be moved here. @philipcmonk, do you remember how e.g. https://github.com/urbit/urbit/issues/139 was migrated?

**Comment by juped** [comment]
I'm in the middle of this already

**Comment by ohAitch** [comment]
Empirically wontfix

**Comment by Fang-** [comment]
Moving issues isn't natively supported by GitHub, but there seems to be third party tooling for it. I can probably find some time tomorrow or over the weekend to do the needed house-cleaning on the urbit/urbit issues.

**Comment by ohAitch** [comment]
There's 100 of them so getting the u3/non-u3 tags up to date might take
some sorting, but if you're up for it that would be great!

**Comment by Fang-** [comment]
Looks like [this](https://github.com/IQAndreas/github-issues-import) was used, and it seems to be the nicest tool around.

It advises to use a dedicated bot account for this, as it becomes the author of all imported issues and their comments. It doesn't seem we have any such account yet. Any objections to me creating one and granting it the relevant permissions? cc @galenwp

**Comment by Fang-** [comment]
Alright, I went over all of them and moved the ones where it was obvious to me the issue didn't lay (even partially) in vere. Some were a bit unclear (ie 828, 832) to me without taking the deep dive, so I left those untouched. For issues where reproduction was trivial, I gave that a shot before moving or closing appropriately.

There are 15 issues that couldn't be moved automatically, the tool failing with:

```
ERROR: There was a problem importing the issues.
422 Unprocessable Entity
DETAILS: Validation Failed
```

These are the yet-unmoved issues:

```
323 352 353 358 359
370 375 407 613 630
653 686 687 711 717
```

I'll have to either dive into the tool's code/dependencies to see what's tripping it up, or just move them over manually.

(Sorry for the notification spam this caused, everybody.)
